src/commands/wallet/sync.rs

```diff
@@ -132,15 +131,12 @@ impl Command {
             // fully-scanned height and the current chain tip.
             #[cfg(feature = "transparent-inputs")]
             for account_id in db_data.get_account_ids()? {
-                let start_height = db_data
-                    .block_fully_scanned()?
-                    .map(|meta| meta.block_height())
-                    .unwrap_or(wallet_birthday);
                 info!(
                     "Refreshing UTXOs for {:?} from height {}",
-                    account_id, start_height,
+                    account_id,
+                    BlockHeight::from(0),
                 );
-                refresh_utxos(params, client, db_data, account_id, start_height).await?;
+                refresh_utxos(params, client, db_data, account_id, BlockHeight::from(0)).await?;
```

Comment:
For large (as in exchange-large) wallets this could be very big (hundreds of thousands of UTXOs fetched each time). I guess that's fine to do in the devtool, but we''ll want to have a think before taking the same approach in Zallet.

Reply:
We might store an account-level "high water mark" that starts at 0, and gets reset to the current chain tip each time UTXOs are fetched for an account.

My intention had originally been to integrate something like this with the retrieval mechanism in https://github.com/zcash/librustzcash/pull/1736 (where I had initially considered this on a per-address basis) but opted not to do that for now.

In Zallet, we'll be in a somewhat different situation: we will be scanning full blocks, not compact blocks, and so we won't need to query for UTXOs in the same fashion.